src/Microsoft.Identity.Client/Platforms/netcore/NetCorePlatformProxy.cs

```diff
@@ -77,8 +77,13 @@ public override string GetBrokerOrRedirectUri(Uri redirectUri)
         }
 
         /// <inheritdoc />
-        public override string GetDefaultRedirectUri(string clientId)
+        public override string GetDefaultRedirectUri(string clientId, bool useRecommendedRedirectURI = false)
         {
+            if (useRecommendedRedirectURI)
+            {
+                return Constants.DefaultNetCoreRedirectUri;
```

Comment:
Why is this the default for netcore?

Reply:
So that the system web view can be used: See https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/wiki/msal-net-4#net-core-now-supports-interactive-authentication

Reply:
I'm not a fan of combining. it's also possible to use the system browser on desktop. IMO, we should have another URI, WithLocalhostRedirectUri or we should specify it by default when using System Browsers on netcore and desktop... 

---
In reply to: [298076066](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1231#discussion_r298076066) [](ancestors = 298076066)

Reply:
Will be address in a later PR